### DISCUSSION TITLE
Knowing when my profile was viewed or needs tuning

### DISCUSSION THREAD

**Discussion by jsuchal** [joemasilotti/railsdevs.com#739] [Ideas]
I don't want to go full linkedin stalker-mode here, but I would actually like to know many times my profile was viewed by a customer or if it was sent to a customer (if that ever happens). 

If my profile is looked at and I am not contacted I would love some guidance on how to tune my profile. Maybe even some stats could be useful. Something like "your profile was viewed X times in the last month, thats more than 50% than the average/median on this site". 

Any other ideas?

**Comment by joemasilotti** [comment]
I'm open to sharing how many times your own profile was viewed. But I worry that that doesn't tell the entire story. A number alone doesn't share much about why someone isn't contacting you.

Other potential stats to include are how many companies viewed your profile and how many companies saw your profile in search results (vs. clicking through). The former might nudge folks to tweak their bio while the latter might help folks tweak their hero.

But I'm not sold that this information is valuable enough on its own. Do you have any ideas how this could prove useful? Said another way, if I told you your profile was viewed 20x in the last month what would you do with that information?
